docs/workloads/task/example.md

```diff
@@ -56,39 +58,57 @@ scikit-learn==0.23.2
 ## Deploy
 
 ```bash
-$ python task.py
+$ cortex deploy
 ```
 
 ## Describe
 
 ```bash
-$ cortex get train_iris
+$ cortex get train-iris
+
+# > endpoint: http://***.elb.us-west-2.amazonaws.com/train-iris
 ```
 
 ## Submit a job
 
+You can submit a job by making a POST request to the Task API's endpoint.
+
+Using `curl`:
+
+```bash
+$ export TASK_API_ENDPOINT=<TASK_API_ENDPOINT>  # e.g. export TASK_API_ENDPOINT=https://***.elb.us-west-2.amazonaws.com/train-iris
+$ export DEST_S3_DIR=<YOUR_S3_DIRECTORY>  # e.g. export DEST_S3_DIR=s3://my-bucket/dir
+
+$ curl $TASK_API_ENDPOINT \
+    -X POST -H "Content-Type: application/json" \
+    -d "{\"config\": {\"dest_s3_dir\": \"$DEST_S3_DIR\"}}"
+# > {"job_id":"69b183ed6bdf3e9b","api_name":"trainer",...}
+```
+
+Or, using Python `requests`:
+
 ```python
 import cortex
 import requests
 
 cx = cortex.client("aws")
```

Comment:
nit: add a comment like "aws is the name of the environment used in this example".